cmd/lambda/accounts/main.go

```diff
@@ -123,6 +138,36 @@ func init() {
 // initConfig configures package-level variables
 // loaded from env vars.
 func initConfig() {
+	cfgBldr := &config.ConfigurationBuilder{}
+	Settings = &accountControllerConfiguration{}
+	if err := cfgBldr.Unmarshal(Settings); err != nil {
+		log.Fatalf("Could not load configuration: %s", err.Error())
+	}
+
+	// load up the values into the various settings...
+	err := cfgBldr.WithEnv("AWS_CURRENT_REGION", "AWS_CURRENT_REGION", "us-east-1").Build()
+	if err != nil {
+		log.Printf("Error: %+v", err)
+	}
+	svcBldr := &config.ServiceBuilder{Config: cfgBldr}
+
+	_, err = svcBldr.
+		// AWS services...
+		WithDynamoDB().
+		WithSTS().
+		WithS3().
+		WithSNS().
+		WithSQS().
+		// DCE services...
+		WithStorageService().
```

Comment:
Seems redundant to be building both `WithS3()` and `WithStorageService()`

Reply:
Yea, it can be considered redundant.  But the DCE services will utilize the AWS services.  StorageService has the ability to read go template files from S3 but it uses the S3 service.